src/components/Table/table-cells/TableCell.scss

```diff
@@ -1,5 +1,23 @@
 @import '../../../utils/index.scss';
 
+.moonstone-tableCellComponent {
+    overflow-y: auto;
+
+    &::-webkit-scrollbar {
+        height: 2px;
```

Comment:
As the height only applies to WebKit browser the behavior is a little weird with other browsers like Firefox. I don't know how and if we can manage that ...

Reply:
I increased it to 4px actually, we can have it it doesn't distort anything in ff. I also added a similar prop for ff but it is still quite wide, covers content although it is semi transparent and you can still see a thing or two.